src/BenchmarkTools.jl

```diff
@@ -10,7 +10,13 @@ if VERSION >= v"0.7.0-DEV.3052"
     using Printf
 end
 
-const BENCHMARKTOOLS_VERSION = v"0.2.2"
+const BENCHMARKTOOLS_VERSION = v"0.3.0"
```

Comment:
@ararslan Note that I think this is a big enough change the we should bump the minor number, but I can remove this from this PR.

Reply:
I think it's fine to have in this PR. Even though this is recorded in the JSON stuff used by Nanosoldier, the version isn't actually used by anything AFAIK. Thanks for the heads up!